docs/configuration/local.md

```diff
@@ -3,13 +3,15 @@
 If running the pipeline in a local environment, we highly recommend using either Docker or Singularity.
```

Comment:
Please drop this file - instead link to this where you need to: https://nf-co.re/usage/local_installation